### ISSUE TITLE
Add light mode suport

### ISSUE THREAD

**Issue by mitchdowney** [podverse/podverse-rn#1550]
We actually have the foundations for light mode already in place, but as a small open source team with day jobs, we decided to drop light mode support because we didn't have the resources to support and maintain two color schemes.

There are so many major initiatives on my plate at this time, I don't realistically see myself having the time to add light mode support any time soon, but I think this could be a great feature for someone who would like to make an open source contribution, without having to learn much about how the code in the Podverse mobile app works.

The bulk of the work is just thoroughly combing through each screen, and toggling light and dark mode on, and setting different colors for each theme as needed. If you run into a tricky part and can't figure it out, I can jump in to assist, or you could just note which element on which screen has the issue and @kreonjr or I can fix it on our own.

If anyone is potentially interested in working on light mode in Podverse, please let me know and I would be happy to help get you started.

**Comment by anibalism** [comment]
Hi there, I just noticed a bug in the Podverse Android app. The bug: When Android is set to dark mode, the color of summary text of the podcast that's being listened to changes from white (in light mode) to gray. 

This makes viewing the summary while you're listening or while the podcast is paused difficult. I was about to report a bug but I'm glad I found this issue first!

Since I basically was doing what you've requested already, I'm willing to fix the issue and also fix the problem with dark mode. 

I'm a bit tech savvy but not a programmer by any means. If you have any other info that could guide me with fixing it I would love to do it. I can start doing it on tuesday. Thanks a lot for this great app!

**Comment by mitchdowney** [comment]
Hi @anibalism, I just tried reproducing the issue on a Pixel 4a, Android 13 device with Dark mode turned on, Podverse v4.10.6, and I'm not seeing the issue. Can you provide a screenshot/s of the issue you're seeing?

![image](https://user-images.githubusercontent.com/2262919/212492268-2ceb1ce1-bfbe-4e34-889d-befcc71206cc.png)

![image](https://user-images.githubusercontent.com/2262919/212492271-f363482c-476c-436d-9aef-a9badf00a6f7.png)

![image](https://user-images.githubusercontent.com/2262919/212492281-77a5aef1-16c7-434e-a7ff-c201aa1c74ee.png)

![image](https://user-images.githubusercontent.com/2262919/212492287-161b7b4c-17f7-419c-aa22-cae218e7018c.png)

**Comment by anibalism** [comment]
Yes of course. Maybe it's related to this phone being a Xiaomi device (Mi 9 Lite, running Android 11).





![Screenshot_2023-01-17-20-30-31-879_fr neamar kiss](https://user-images.githubusercontent.com/2481013/212994820-b0112771-56af-4cad-886b-8af7fe3c891f.jpg)
![Screenshot_2023-01-17-20-31-30-519_com podverse fdroid](https://user-images.githubusercontent.com/2481013/212994823-50951716-edd3-4cde-9651-8eaf09043341.jpg)
![Screenshot_2023-01-17-20-31-41-952_com podverse fdroid](https://user-images.githubusercontent.com/2481013/212994826-b5fc97cc-2c7f-4a31-91a4-25b2b1f506c2.jpg)
![Screenshot_2023-01-17-20-32-00-554_com podverse fdroid](https://user-images.githubusercontent.com/2481013/212994832-2935cd5f-38fb-4c7c-9e7e-3aedde87e219.jpg)
![Screenshot_2023-01-17-20-32-06-741_com podverse fdroid](https://user-images.githubusercontent.com/2481013/212994835-60654dc3-e597-4232-98ec-92db86000345.jpg)

**Comment by mitchdowney** [comment]
Thanks for the screenshots. Hmm this is really unexpected...I just checked the code to see if we are doing anything that could cause text style to load differently, but it looks like we just have it hardcoded to be white all the time.

Something else I see in your screenshots is that the player progress slider has a dark gray position indicator, when that should be white too. It looks like it might be the same dark gray as the text. It seems like your device may be selectively trying to convert text colors on some UI elements, but not others? Is it possible you have any "inverse colors" settings on or something?

It looks like you have Reading Mode turned on, although I don't think that should affect colors in apps.

Unfortunately I'm pretty stumped at the moment. It's strange that some white elements on your screen are getting converted, but not all of them. Until I have a way to reproduce this issue this might be tricky for me to fix.

**Comment by Parameter15** [comment]
I am not a fan of dark mode, so having a light mode would be really nice. =)

**Comment by AmanNegi** [comment]
This sounds interesting, I will try to help you with this. I am familiar with Flutter & React and a little but of React Native. For now, I will test the app and explore the codebase. If you have any tips/suggestions for me to get started please let me know.

**Comment by mitchdowney** [comment]
We are archiving all Github issues and PRs that pertain to the previous generation Podverse. The Podverse dev team is currently 100% focused on the next generation app, and will be only working on next gen Github issues going forward.

If you would like to see a work-in-progress demo of the upcoming Podverse website, please visit https://alpha.podverse.fm. Our tentative release date is Spring 2026.

If you're a developer who may be interested in contributing to Podverse Next-Gen, please see our [Local Dev Setup documentation](https://github.com/podverse/podverse-ops/blob/v5-develop/dev/local-dev-setup.md). We can also be reached through [Matrix or Discord](https://podverse.fm/contact).

I'm sorry for the abrupt announcement, but thank you for your bug reports and feature requests.
